### PR TITLE
Arm

### PR DESCRIPTION
**Pull request by rmheuer**
Works in simulator, will need arm to be built and changing constants to work on robot

### CODE REVIEW

File: Pathfinding/src/main/java/com/swrobotics/pathfinding/core/FinderThread.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+package com.swrobotics.pathfinding.core;
+
+import com.swrobotics.pathfinding.core.finder.Pathfinder;
+
+import java.util.List;
+
+public abstract class FinderThread<P> extends Thread {
```
Comment (foehammer127) [inline Pathfinding/src/main/java/com/swrobotics/pathfinding/core/FinderThread.java:7]: ugh threading

**Review by foehammer127 — COMMENTED** [review]
noice

File: Pathfinding/src/main/java/com/swrobotics/pathfinding/core/FinderThread.java (hunk 1 above)
Comment (rmheuer) [reply Pathfinding/src/main/java/com/swrobotics/pathfinding/core/FinderThread.java:7]: Might as well use all the cpu we have

**Comment by rmheuer** [comment]
We should probably wait to merge this until we have built the physical arm

**Comment by rmheuer** [comment]
Will require updating Pathfinding jar on RPi

**Review by foehammer127 — APPROVED** [review]

**Pull request merged**